src/internal/operations/inject/inject.go

```diff
@@ -116,4 +113,34 @@ type (
 	RepoMaintenancer interface {
 		RepoMaintenance(ctx context.Context, opts repository.Maintenance) error
 	}
+
+	// ServiceHandler contains the set of functions required to implement all
+	// service-specific functionality for backups, restores, and exports.
+	ServiceHandler interface {
+		ExportConsumer
+	}
+
+	ToServiceHandler interface {
+		NewServiceHandler(
+			opts control.Options,
+			service path.ServiceType,
+		) ServiceHandler
+	}
 )
+
+var _ ServiceHandler = &BaseServiceHandler{}
+
+type BaseServiceHandler struct{}
+
+func (h BaseServiceHandler) ProduceExportCollections(
+	ctx context.Context,
+	backupVersion int,
+	exportCfg control.ExportConfig,
+	dcs []data.RestoreCollection,
+	stats *data.ExportStats,
+	errs *fault.Bus,
+) ([]export.Collectioner, error) {
+	return nil, clues.New("not implemented")
```

Comment:
I'd prefer to avoid implementations and interface fulfillment within the inject package, if possible.  Better for us to let it only declare structures and interfaces used elsewhere.

Per my other comment, this doesn't seem like it has much purpose outside of delaying an error return.  Better off, imo, to remove it altogether if possible.

Reply:
we're probably going to want something like this in the long run, at least to help with bootstrapping and/or interfaces changes going forward. There was a similar debate in kopia a bit back about how best to expand the blob storage interface. Turns out if all things embed a "default" handler then you can add no-op implementations and get compliance immediately. Things that need overrides can override it

We can remove it from this PR, but there's probably a good chance we'll end up with it again

Reply:
mmkay.  Lets keep it.  Though, per the inject package, maybe it could live in pkg/export?  Or collections.  Or whatevs, its' all good.  Also, how's about a TODO to remove it when migration is complete?